examples/getting_started/README.md

```diff
@@ -0,0 +1,596 @@
+
+And, last, let's add a test suite to .kitchen.yml
+
+.kitchen.yml
+```
+---
+driver:
+  name: terraform
+
+provisioner:
+  name: terraform
+  variable_files:
+    - testing.tfvars
+
+platforms:
+  - name: ubuntu
+
+transport:
+  name: ssh
+  ssh_key: ~/path/to/your/private/aws/key.pem
+
+verifier:
+  name: terraform
+  format: doc
+  groups:
+  - name: default
+    tests:
+      - operating_system
+    hostnames: public_dns
+    username: ubuntu
+
+suites:
+  - name: default
+```
+
+Save and close the file.
+
+## Writing a test
+
+Now, we have some work to do on our workstation.  First, let's set up
+an inspec directory within our test directories
+
+```
+  $ mkdir -p test/integration/default/controls
+```
+
+This will be our inspec group of tests.  Now we need to provide a yml
+file with the name of that group within the group directory.  Go ahead
+and create this:
+
+```
+  $ vim test/integration/default/inspec.yml
+```
+
+And add this content
+
+test/integration/default/inspec.yml
+```
+---
+name: default
+```
+
+Save and close the file.
+
+Now, just one more bit of housekeeping.  Our .kitchen.yml is expecting
+an output of our test kitchen instances' hostnames within the output
+variable public_dns. In order to have a hostname within that public_dns
+variable, we need to create an EC2 instance.
+
+First, open up the main config file.
+
+```
+  $ vim main.tf
+```
+
+And add this content:
+
+main.tf
+```
+provider "aws" {
+  access_key = "${var.access_key}"
+  secret_key = "${var.secret_key}"
+  region = "${var.region}"
+}
+
+resource "aws_instance" "example" {
+  ami           = "${var.ami}"
+  instance_type = "${var.instance_type}"
+  key_name      = "${var.key_name}"
+}
+```
+
+Save and close the file.  Notice that we used some variable values
+there?  We need to add these into our variables.tf file.
+
+Go ahead and create it.
+
+```
+  $ vim variables.tf
+```
+
+And add in this content:
+
+variables.tf
+```
+variable "access_key" {}
+variable "secret_key" {}
+variable "key_name" {}
+variable "region" {}
+variable "ami" {}
+variable "instance_type" {}
+```
+
+And, finally, we need to add in some values for these variables.
+
+Open up your testing.tfvars file
+
+```
+  $ vim testing.tfvars
+```
+
+And add in this content (substitute in the appropriate values for your
+AWS account, region, etc.)
+
+testing.tfvars
+```
+access_key = "my_aws_access_key"
+secret_key = "my_aws_secret_key"
+key_name = "my_aws_key_pair_name"
+region = "us-east-1"
+ami = "ami-fce3c696"
+instance_type = "t2.medium"
+```
+
+Save and close the file.
+
+Finally, let's define that output that will be expected by our
+.kitchen.yml.
+
+Go ahead and open up your output.tf file.
+
+```
+  $ vim output.tf
+```
+
+And add this content
+
+```
+output "public_dns" {
+  value = "${aws_instance.example.public_dns}"
+}
+```
+
+Ok!  Now we're ready to finally create some test kitchen instances!
+
+Go ahead and run
+```
+  $ bundle exec kitchen converge
+```
```

Comment:
Thanks for letting me know, I think this is because you may not be using HVM type image, as documented here.  I will add in info about what to do with this error :)